libbpf-cargo/src/btf/btf.rs

```diff
@@ -97,7 +103,7 @@ impl<'a> Btf<'a> {
             types: vec![BtfType::Void],
             ptr_size: ptr_size as u32,
             string_table: str_data,
-            bpf_obj,
+            _bpf_obj: bpf_obj,
```

Comment:
it's actually weird and wrong for BTF object to contain bpf_object inside. But I don't think we need to do that here if we just make our own copy of raw_data and keep *that* in Btf. WDYT?

Reply:
I've adjusted the code. Let me know if that's not what you had in mind.